pkg/rest/request.go

```diff
@@ -19,8 +19,10 @@ package rest
 import (
 	"bytes"
 	"fmt"
+	version2 "github.com/dynatrace/dynatrace-configuration-as-code/pkg/version"
```

Comment:
nitpick: Why are you using an alias?

Reply:
because I copied the header line from pkg/client/transport and didn't notice the import name that used :D 
version2 seems necessary there as well tho

Reply:
addressed